enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -354,14 +356,7 @@ func NewContainer(ctx context.Context, env environment.Env, imageCacheAuth *cont
 	}
 
 	c := &FirecrackerContainer{
-		vmConfig: &fcpb.VMConfiguration{
-			NumCpus:           opts.NumCPUs,
-			MemSizeMb:         opts.MemSizeMB,
-			ScratchDiskSizeMb: opts.ScratchDiskSizeMB,
-			EnableNetworking:  opts.EnableNetworking,
-			InitDockerd:       opts.InitDockerd,
-			DebugMode:         opts.DebugMode,
-		},
+		vmConfig:           opts.VMConfiguration,
```

Comment:
Is it correct that this will be nil if opts.SavedState is set?

To prevent confusion where opts.VMConfiguration can be optionally set, should we just always populate it from opts.SavedState?